src/Core/src/[Microsoft.Maui.Graphics]/Linux/ColorExtensions.cs

```diff
@@ -0,0 +1,42 @@
+namespace Microsoft.Maui.Graphics.Native.Gtk
```

Comment:
Will ask about this. We have different options:
- Create a Linux specific Microsoft.Maui.Graphics library here
- Push the Linux changes directly to the Microsoft.Maui.Graphics repo.

Reply:
please, your opinion to "Linux" versus "Gtk"; see: https://github.com/jsuarezruiz/maui-linux/issues/6

second options would be better. 

i'm planing to implement a Microsoft.Maui.Graphics Gtk NativeCanvas (if no others want to do it)

Reply:
is there any point in running Gtk on Windows or macOS, though, especially if we're talking within the context of Maui? Because Maui's Windows and macOS support is a first-class citizen, while Gtk's first-class citizen is Linux (I myself found many Gtk bus in Windows and macOS; so many, that we should discourage its use).

Reply:
s/bus/bugs

Reply:
BTW guys you might want to get some inspiration from here (https://github.com/xamarin/Xamarin.Forms/pull/14235) to implement Shapes API, a PR for XF that I just opened.

Reply:
@knocte thanks for the hint!

> to get some inspiration from here (xamarin/Xamarin.Forms#14235) 

but for implementing maui - shapes, i would propose to have pure maui-renderers using https://github.com/dotnet/Microsoft.Maui.Graphics/blob/main/src/Microsoft.Maui.Graphics/ICanvas.cs for all platforms

> is there any point in running Gtk on Windows or macOS

it could be a point having a Maui-platform KDE / Qt on linux, so changing "linux" -> "Gtk" makes sense anyway. None of the code is linux-specific by now, but Gtk-specific

Reply:
> it could be a point having a Maui-platform KDE / Qt on linux, 

It could, but QtSharp has been in alpha development for years (if not more than a decade) and no project uses it. It must be real hard to bind C++ from C#... So I would only rename 'linux' to 'gtk' when QtSharp+Maui really happens for real (which may be never).